cmd/genautocomplete/genautocomplete_bash.go

```diff
@@ -18,20 +18,31 @@ var bashCommandDefinition = &cobra.Command{
 	Long: `
 Generates a bash shell autocompletion script for rclone.
 
-This writes to /etc/bash_completion.d/rclone by default so will
-probably need to be run with sudo or as root, e.g.
+By default, when run without any arguments, 
 
-    sudo rclone genautocomplete bash
+    rclone genautocomplete bash
 
-Logout and login again to use the autocompletion scripts, or source
-them directly
+the generated script will be written to
 
-    . /etc/bash_completion
+/etc/bash_completion.d/rclone
 
-If you supply a command line argument the script will be written
-there.
+and so will probably need to be run as root, or with sudo.
```

Comment:
I think this needs a little more clarification, maybe

    ...and so rclone will probably need to be run as root...

As I wasn't sure whether it was the generated script or rclone that needed running as root reading that.